pyproject.toml

```diff
@@ -95,7 +99,8 @@ addopts = """
     --ignore=src/blueapi/startup
     """
 # https://iscinumpy.gitlab.io/post/bound-version-constraints/#watch-for-warnings
-filterwarnings = ["error", "ignore::DeprecationWarning"]
+# Unignore UserWarning after Pydantic warning removed from bluesky/bluesky and release
```

Comment:
https://github.com/bluesky/bluesky/pull/1867

+ a release of bluesky

Reply:
Should: Include issue link in a comment in the file